frontend/simple/views/components/Tooltip.vue

```diff
@@ -0,0 +1,106 @@
+<template>
+  <span @mouseenter="show" @mouseleave="hide">
```

Comment:
Note: when needed, this component can be improved with more options:
- Click instead of hover,
- More directions
- Controlled tooltip from outside

For now I've just did the needed API for this particular task but I thing it's pretty easy to scale it.